db/schema.rb

```diff
@@ -56,7 +56,7 @@
     t.datetime "updated_at", null: false
     t.string "file_file_name"
     t.string "file_content_type"
-    t.integer "file_file_size"
+    t.bigint "file_file_size"
```

Comment:
Is ok to change schema without migration?

Reply:
Yes